### PR TITLE
[FIX] Various Finnish invoice adjustments

### PR DESCRIPTION
**Pull request by n17**
- Made black line gray, so that it doesn't stick out
- Increased "Total" row text size in taxes section
- Improved text alignment
- Link to task: https://odoo.web-veistamo.fi/web#id=11831&view_type=form&model=project.task&action=201&menu_id=192
- [Finnish_invoice_before.pdf](https://github.com/avoinsystems/avoinsystems-addons/files/3306284/Finnish_invoice_before.pdf)
- [Finnish_invoice_after.pdf](https://github.com/avoinsystems/avoinsystems-addons/files/3306283/Finnish_invoice_after.pdf)

### CODE REVIEW

**Review by mlaitinen — APPROVED** [review]

**Comment by aisopuro** [comment]
There are no other reviewers being waited on. Is this still waiting for a second review or is it ready to be merged?

**Comment by mlaitinen** [comment]
> There are no other reviewers being waited on. Is this still waiting for a second review or is it ready to be merged?

I have an pending query to Svante about this in our Odoo. And he's on holiday this month.

We could just merge this and discuss the improvements when Svante is back.

**Comment by aisopuro** [comment]
OK. I have no particular strong feelings about what should be done, mainly I just wanted the reason to be stated on this PR for clarity.

Do whatever you feel is best. I would probably merge this if the query to Svante is of the form "do you think this can still be improved", but I'd wait on the merge if the question is like "is this OK now".

**Pull request merged**
